packages/components/src/css/component/Typography.module.scss

```
@@ -0,0 +1,150 @@
+
+    // Table, matching the FluxTable component.
+    table#{$guard} {
+        width: 100%;
+        border-collapse: collapse;
+        text-align: start;
+    }
+
+    :where(th, td)#{$guard} {
+        padding: 12px 15px;
+        text-align: start;
+        word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)